captum/insights/frontend/src/App.js

```diff
@@ -461,6 +462,30 @@ function Visualizations(props) {
   );
 }
 
+class AppBase extends React.Component {
+  componentDidMount() {
+    this.props.fetchInit();
+  }
+
+  render() {
+    return (
+      <div className={styles.app}>
+        <Header />
+        <FilterContainer
+          fetchData={this.props.fetchData}
+          config={this.props.config}
+          key={this.props.config}
+        />
+        <Visualizations
+          data={this.props.data}
+          loading={this.props.loading}
+          onTargetClick={this.props.onTargetClick}
+        />
+      </div>
+    );
+  }
+}
+
 class App extends React.Component {
```

Comment:
Can we move `App` to a separate file, maybe `WebApp` to make it clear that its a separate backend implementation?